linux/Jenkinsfile

```diff
@@ -532,7 +532,6 @@ def uploadDebArtifacts(String buildArch, String Version) {
             "trixie",   // Debian/13
             "bookworm", // Debian/12
             "bullseye", // Debian/11
-            "buster",   // Debian/10
```

Comment:
I think this should be left in the jenkinsfile, as its just used to publish packages in the artifactory repo.

Reply:
Fixed :)